lib/KBox/src/KMessage.h

```diff
@@ -157,20 +164,20 @@ class IMUMessage: public KMessage {
     void accept(KVisitor &v) const {
       v.visit(*this);
     };
-
+    
```

Comment:
Not really certain what you are getting at here but have tried to make spacing consistent for everything on this page.